ciao-controller/internal/datastore/datastore_test.go

```diff
@@ -797,45 +797,45 @@ func TestGetBatchFrameSummary(t *testing.T) {
 
 	err := ds.HandleTraceReport(trace)
 	if err != nil {
-		t.Error(err)
+		t.Fatal(err)
 	}
 
 	_, err = ds.db.getBatchFrameSummary()
 	if err != nil {
-		t.Errorf(err.Error())
+		t.Fatalf(err.Error())
```

Comment:
Fatalf takes a string format - Fatal does not. As to why - that is a result of different people writing the unit tests and copy and paste. making the usage consistent would be a different PR I believe. This PR is just to prevent the tests from continuing when they should halt.